mlrun/api/utils/scheduler.py

```diff
@@ -367,8 +400,33 @@ async def submit_run_wrapper(scheduled_object, schedule_name):
         # otherwise all runs will have the same uid
         scheduled_object.get("task", {}).get("metadata", {}).pop("uid", None)
 
+        if "task" in scheduled_object and "metadata" in scheduled_object["task"]:
+            scheduled_object["task"]["metadata"].setdefault("labels", {})
+            scheduled_object["task"]["metadata"]["labels"][
+                schemas.constants.LabelNames.schedule_name
+            ] = schedule_name
+
         db_session = create_session()
 
+        schedule_project = (
+            scheduled_object.get("task", {}).get("metadata", {}).get("project", None)
+        )
```

Comment:
I think this should come from outside (just like the `schedule_name`). The scheduled object (Run) project can come from several sources (and when all of them are empty default project is used)